biojava-structure/src/main/java/org/biojava/nbio/structure/cath/CathInstallation.java

```diff
@@ -636,6 +636,7 @@ private void parseCathDomall(BufferedReader bufferedReader) throws IOException{
 
 	protected void downloadFileFromRemote(URL remoteURL, File localFile) throws IOException{
 //        System.out.println("downloading " + remoteURL + " to: " + localFile);
+		LOGGER.info("Downloaded file {} to local file {}", remoteURL, localFile);
```

Comment:
Shouldn't this be "Downloading"? It isn't downloaded at this point yet

Reply:
fixed